util/common.py

```diff
@@ -51,8 +60,11 @@ def getQCFlagMeanings(cls):
 
     @classmethod
     def isValidQCFlag(cls, flag):
-        return flag in QartodFlags.getValidQCFlags()
+        return flag in cls.getValidQCFlags()
 
+def is_qc_parameter(param):
+    #return any(quality_variable in param for quality_variable in [QC_EXECUTED, QC_RESULTS, QARTOD_PRIMARY, QARTOD_SECONDARY])
```

Comment:
this function always returns true? what's the point of it then?

Reply:
I believe the intent was to use the code that's commented here, right?

Reply:
Tim is correct. I had commented the line out when trying to debug an error and failed to restore it. Fixing.